features/nanostack/FEATURE_NANOSTACK/targets/TARGET_SL_RAIL/NanostackRfPhyEfr32.cpp

```diff
@@ -254,6 +377,12 @@ static int8_t rf_device_register(void)
         radio_state = RADIO_INITING;
     }
 
+#ifdef MBED_CONF_RTOS_PRESENT
```

Comment:
Does this make sense if this file already defines a thread above (line 69), thus assuming that without this conf macro present, this file would not compile anyway?